api/docs/v2/storefront/index.yaml

```diff
@@ -94,18 +94,6 @@ paths:
                           state_code: NY
         '403':
           $ref: '#/components/responses/403Forbidden'
-          description: ''
```

Comment:
@MatthewKennedy why are we removing this one?

Reply:
According to Swagger:

`Sibling values alongside $refs are ignored. To add properties to a $ref, wrap the $ref into allOf, or move the extra properties into the referenced definition (if applicable).`

Reply:
As far as I can tell from the output `$ref: '#/components/responses/403Forbidden'` is taking care of the description.